extensions/fairifier-rdf-extension/dtls/fairifier/RdfSkeletonTransformer.java

```diff
@@ -0,0 +1,9 @@
+package org.dtls.fairifier;
+
+
+public interface RdfSkeletonTransformer {
+    public void transform(String model);
+    public String getModelAsJsonString();
+    public SkeletonMetadata getSkeletonMetadata();
+    public void setModelMetadata(SkeletonMetadata metadata );
+}
```

Comment:
Is this interface according to the design we discussed last sprint? I seem to recall a different idea for the transformer design, but maybe I'm off.